### PR TITLE
Update apiref for blockdevs

### PR DESCRIPTION
**Pull request by agrover**
Still need to work on blockdev states.

### CODE REVIEW

**Comment by agrover** [comment]
Refers to content added to the design doc in #62.

**Comment by agrover** [comment]
screenshot from 2017-08-31 15-23-55

File: docs/dbus/DBusAPIReference.lyx [hunk 1]
```diff
@@ -405,9 +405,9 @@ devices device nodes of devices to form the pool.
 
 \end_deeper
 \begin_layout Description
-Returns: the object path of the constructed pool and the device nodes of
- the blockdevs added to the pool.
- Signature: (oas)qs.
+Returns: the object path of the constructed pool and the object path and
+ device nodes of the blockdevs added to the pool.
+ Signature: oa(os)qs.
```
Comment (mulkieran) [inline docs/dbus/DBusAPIReference.lyx]: Maybe better just to have object path, and to drop devnode part for this and the one below.

**Comment by mulkieran** [comment]
State's signature is "s"?

File: docs/dbus/DBusAPIReference.lyx (hunk 1 above)
Comment (agrover) [reply docs/dbus/DBusAPIReference.lyx]: sounds good.

**Comment by agrover** [comment]
Added missing signature for State property. Documented that methods with data return a triple, with the data in the first element, and possibly being a container type. Incremented version to 0.2. Drop devnode from CreatePool and AddDevs return data in favor of object paths.

**Comment by agrover** [comment]
@mulkieran any more review comments?

**Pull request merged**